mojoshader_d3d11.c

```diff
@@ -0,0 +1,770 @@
+/**
+ * MojoShader; generate shader programs from bytecode of compiled
+ *  Direct3D shaders.
+ *
+ * Please see the file LICENSE.txt in the source's root directory.
+ *
+ *  This file written by Ryan C. Gordon.
+ */
+
+#ifdef _WIN32
```

Comment:
This _WIN32 check can be removed - the only time this file should be compiled is either on Windows or with DXVK-Native present.